src/appMain/CMakeLists.txt

```diff
@@ -35,9 +35,10 @@ find_package(SDLOpenSSL REQUIRED)
 include_directories(
   ${COMPONENTS_DIR}/protocol_handler/include
   ${COMPONENTS_DIR}/application_manager/include
+  ${COMPONENTS_DIR}/can_cooperation/include
   ${COMPONENTS_DIR}/formatters/include
   ${COMPONENTS_DIR}/transport_manager/include
-  ${COMPONENTS_DIR}/security_manager/include
+  ${SecurityManagerIncludeDir}
```

Comment:
@dev-gh removed in https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/69b0106c66b759cb7e1c5b56cd36e7c35b1a4867